source-kafka/src/main.rs

```diff
@@ -1,8 +1,23 @@
 use structopt::StructOpt;
 
-use source_kafka::connector::Command;
+use tracing_subscriber::fmt::format::FmtSpan;
+use tracing_subscriber::EnvFilter;
 
+use source_kafka::connector::Command;
 fn main() -> color_eyre::Result<()> {
+    color_eyre::install()?;
```

Comment:
optional nit: If there's a non-color version of eyre, then it might be worth using that instead. Just thinking that we're very unlikely to run connectors where stdout/stderr is a tty, and a non-color version might  have fewer dependencies and compile faster.